multiversx/mx-chain-go
# Governance proposal lose fee

## Reasoning behind the pull request
Added a fee when proposal is lost

## Proposed changes
Not sending back the full amount of deposited tokens, fee remains in contract, can be claim by contract owner.

## Testing procedure
Start a proposal and make it one which is not won (not enough quorum - for example). You will receive less eGLD back then deposited. Check the fee to be equal to the one in config.

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
